app/components/markdown/markdown_table/markdown_table.js

```diff
@@ -122,22 +154,22 @@ export default class MarkdownTable extends React.PureComponent {
                         changeOpacity(this.props.theme.centerChannelColor, 0.0),
                         changeOpacity(this.props.theme.centerChannelColor, 0.1),
                     ]}
-                    style={style.moreBelow}
+                    style={[style.moreBelow, {width: renderAsFlex ? '100%' : tableWidth}]}
                 />
             );
         }
 
         return (
             <TouchableOpacity onPress={this.handlePress}>
                 <ScrollView
-                    contentContainerStyle={{width: this.getTableWidth()}}
+                    contentContainerStyle={{width: '100%'}}
```

Comment:
You might be able to remove this prop since I think this is the default value for the prop. If not though, would you mind adding it to the `StyleSheet` as well?